client/src/App.js

```diff
@@ -22,16 +23,36 @@ function App() {
 
   useEffect(() => {
     userGet()(dispatch);
+    return () => dispatch("FINISH_LOAD");
   }, [dispatch]);
-  console.log(state);
+  console.log((() => !state?.user?.id)());
```

Comment:
I feel like we shouldn't even be consoling logging this tbh

Reply:
forgot to remove